Xamarin.PropertyEditing.Windows.Standalone/MainWindow.xaml

```diff
@@ -18,8 +18,8 @@
 			<RowDefinition />
 		</Grid.RowDefinitions>
 
-		<Button Grid.Row="0" Grid.Column="0" Click="Button_Click">Select me</Button>
-		<Button Grid.Row="1" Grid.Column="0" Click="Button_Click">Select me 2</Button>
+		<local:MockedWpfButton Grid.Row="0" Grid.Column="0" Click="Button_Click">Select me (Mock)</local:MockedWpfButton>
```

Comment:
Let's move to 3 buttons, two of the mocked type so we can still test same properties for multi-select.

Reply:
I'm thinking of adding a simpler mocked type that has only one property of each supported type. We could have two buttons of that. That would be simple mocked 1, simple mocked 2, full platform-specific mocked, reflection-based. What do you think?

Reply:
That seems fine, that's kind of what I intended originally, but this was fine too.

Reply:
It's probably useful in the long run to keep at least one representation of a full control for accuracy and to get a good idea of the actual experience editing real controls, but as we work day to day on supporting more property types, having a more distraction-free experience will save us some time. I'll add it.